event_registration_partner_unique/__openerp__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Unique Partner per Event",
+    "summary": "Enforces 1 registration per partner and event",
+    "version": "8.0.1.0.0",
+    "category": "Events",
+    "website": "http://www.antiun.com",
+    "author": "Antiun Ingeniería S.L., Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "pre_init_hook": "pre_init_hook",
+    "depends": [
+        "event",
```

Comment:
That module just adds a shortcut. See https://github.com/OCA/event/pull/38#issuecomment-190296383